_ont/ont-huawei-ma5671a-root.md

```diff
@@ -12,24 +12,24 @@ layout: default
 | 3.3V                  | red                    | pin #15 and #16            |
 | TX                    | orange                 | pin #2                     |
 | RX                    | yellow                 | pin #7                     |
-| GND                   | green                  | pin #10                    |
+| GND                   | green                  | pin #14                    |
```

Comment:
It would be good to add a warning to inform the user to try PIN 10 or other GND PINs if it doesn't work with 14.
Unfortunately, there are poorly assembled sticks that cause problems

Reply:
it would also be necessary to edit all the other pages of the MA

Reply:
I'll handle the parent page